loggy/README.md

```diff
@@ -1,66 +1,48 @@
 # Loggy
-Customizable logger for dart.
+![Loggy icon][loggy_image]
```

Comment:
The image does not get displayed.

<img width="1244" alt="Screenshot 2020-12-02 at 09 46 28" src="https://user-images.githubusercontent.com/18558083/100849976-6b455c80-3483-11eb-866e-a1204626aed0.png">

Reply:
Yeah I know that, that will show once we merge PR because image is using link to get from master branch:
`https://github.com/infinum/floggy/raw/master/assets/loggy_image.png`